Revise and expand pediatric paragraph in clinical presentation (#476)
greenelab/covid19-review: content/02.introduction.md

Outcomes in patients that are hospitalized have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032; @doi:10.1001/jama.2020.2648; @doi:10.1001/jama.2020.4031].

The results of COVID-19 infection can vary greatly among pediatric patients as well as adults. 
[@doi:10.1001/jamapediatrs.2020.1467] reviewed studies on children infected with COVID-19 during the early months of COVID-19 in China.
Most of the children reviewed had mild symptoms such as fever, dry cough, fatigue, nasal congestion and/or runny nose or were asymptomatic.
Gastrointestinal symptoms such as vomiting or diarrhea were occasionally reported.
Radiologic findings were not always mentioned, but when mentioned they included bronchial thickening, ground-glass opacities, and/or inflammatory lesions. Neurological symptoms have also been reported [@doi:10.1001/jamaneurol.2020.2687].
Generally, most cases of COVID-19 in children are not severe.  

> However, serious complications, and in rare cases deaths, have occurred
> [@url: https://www.cnn.com/2020/04/21/us/detroit-girl-dies-coronavirus/index.html; @doi:10.15585/mmwr.mm6914e4.]. 

Reviewer: ```suggestion
However, serious complications, and in rare cases deaths, have occurred [@https://www.cnn.com/2020/04/21/us/detroit-girl-dies-coronavirus/index.html; @doi:10.15585/mmwr.mm6914e4]. 
```

Using the CNN story to support this story seems a little anecdotal.  What about replacing the CNN reference with this systematic review:
COVID-19 in 7780 pediatric patients: A systematic review
https://doi.org/10.1016/j.eclinm.2020.100433

Author: I agree!  I was not very happy about citing a TV news station in an academic paper, but couldn't quite decide on anything better.  I will make the change.

Individual outcomes have varied significantly, and reported hospitalization rates have been wide-ranging based on age and location, with various underlying health conditions being disproportionately reported [@doi:10.15585/mmwr.mm6915e3].
Outcomes in patients that are hospitalized have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032; @doi:10.1001/jama.2020.2648; @doi:10.1001/jama.2020.4031].

> The results of COVID-19 infection can vary greatly among pediatric patients as well as adults. 

Author: I think that is a good improvement, although it might be better to say "and perhaps can" instead of "and can," because we don't know for sure that the MIS/C syndrome doesn't happen in adults too.

[@doi:10.1001/jamapediatrs.2020.1467] reviewed studies on children infected with COVID-19 during the early months of COVID-19 in China.
Most of the children reviewed had mild symptoms such as fever, dry cough, fatigue, nasal congestion and/or runny nose or were asymptomatic.
Gastrointestinal symptoms such as vomiting or diarrhea were occasionally reported.

> Radiologic findings were not always mentioned, but when mentioned they included bronchial thickening, ground-glass opacities, and/or inflammatory lesions. Neurological symptoms have also been reported [@doi:10.1001/jamaneurol.2020.2687].

Author: The radiological sentence was from @scapone01, but I will check the paper.  The paper might be a case series (compilation of individual case studies).

Generally, most cases of COVID-19 in children are not severe.  

> However, serious complications, and in rare cases deaths, have occurred
> [@doi:10.1016/j.eclinm.2020.100433]. 

Reviewer 2: ```suggestion
However, serious complications and, in rare cases, death have occurred [@doi:10.1016/j.eclinm.2020.100433]. 
```

> Of particular interest, children have occasionally experienced a serious inflammatory syndrome after COVID-19 infection, similar in some respects to Kawasaki disease or to Kawasaki disease shock syndrome [@doi:10.1093/jpids/piaa069;@doi:10.1001/jama.2020.10369; @doi:10.1016/j.ajem.2020.05.117].

Reviewer 2: ```suggestion
Of particular interest, children have occasionally experienced a serious inflammatory syndrome after COVID-19 infection, similar in some respects to Kawasaki disease or to Kawasaki disease shock syndrome [@doi:10.1093/jpids/piaa069; @doi:10.1001/jama.2020.10369; @doi:10.1016/j.ajem.2020.05.117].
```

Individual outcomes have varied significantly, and reported hospitalization rates have been wide-ranging based on age and location, with various underlying health conditions being disproportionately reported [@doi:10.15585/mmwr.mm6915e3].
Outcomes in patients that are hospitalized have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032; @doi:10.1001/jama.2020.2648; @doi:10.1001/jama.2020.4031].

> The presentation of COVID-19 infection can vary greatly among pediatric patients, and in some cases they might manifest in distinct ways from COVID-19 in adults. 

Reviewer 2: ```suggestion
The presentation of COVID-19 infection can vary greatly among pediatric patients and in some cases manifests in distinct ways from COVID-19 in adults. 
```

> [@doi:10.1001/jamapediatrics.2020.1467] reviewed studies on children infected with COVID-19 during the early months of COVID-19 in China.

Reviewer 2: ```suggestion
[@doi:10.1001/jamapediatrics.2020.1467] reviewed 17 studies on children infected with COVID-19 during the early months of the COVID-19 epidemic in China and one study from Singapore.
```
I'm not sure if we can say they were in China since one was in Singapore -- but 17/18 were in China, so I don't know what the best way to mention the single Singapore study would be!

> Most of the children reviewed had mild symptoms such as fever, dry cough, fatigue, nasal congestion and/or runny nose or were asymptomatic.

Reviewer 2: ```suggestion
Of the more than a thousand cases described, the most commonly reports were for mild symptoms such as fever, dry cough, fatigue, nasal congestion and/or runny nose, while three children were reported to be asymptomatic.
Severe lower respiratory infection was described in only one of the pediatric cases reviewed.
```
The specific mention I found in the article says that these were "frequent clinical manifestations" -- but I might be missing something that supports the "most of the children" phrasing?
I was also thinking it might be worth adding a mention that they did find one severe case.

Gastrointestinal symptoms such as vomiting or diarrhea were occasionally reported.
Radiologic findings were not always reported in the case studies reviewed by [@doi:10.1001/jamapediatrics.2020.1467], but when they were mentioned they included bronchial thickening, ground-glass opacities, and/or inflammatory lesions.
Neurological symptoms have also been reported [@doi:10.1001/jamaneurol.2020.2687].

> Generally, most cases of COVID-19 in children are not severe.  

Reviewer 2: ```suggestion

These analyses indicate that generally, most pediatric cases of COVID-19 are not severe.  
```
